tests/Type/data/model-factories.php

```diff
@@ -7,8 +7,7 @@
 
 use function PHPStan\Testing\assertType;
 
-function doFoo(): void
-{
+function (?int $foo) {
```

Comment:
Isn't this a syntax error? 🤔 Doesn't it need a name? Even if it is valid, I'd rather have a `dummy` name.

Reply:
No syntax error, it's just an anonymous closure. PHPStan does this with their files:

https://github.com/phpstan/phpstan-src/blob/6026869f3eec4623a0be812ef69e1dc27a430d4e/tests/PHPStan/Analyser/data/bug-4339.php#L7

Personally I'd rather just have an anonymous closure instead of a named function like `doFoo` or `dummy`. You can also run into issues with namespace clashes with that approach.

Reply:
That's why we have unique namespace for each data file already. And for readability I'd rather have a dummy name.

Reply:
done